var/spack/repos/builtin/packages/mummer/package.py

```diff
@@ -34,22 +34,34 @@ class Mummer(Package):
     version('3.23', 'f2422b3d2638dba4baedb71b1acdffa2')
 
     depends_on('gnuplot')
+    depends_on('perl')
```

Comment:
This should probably be a build or build/run dependency unless it links to the Perl libraries.